.github/actions/composer/composer/install/action.yaml

```diff
@@ -0,0 +1,22 @@
+# https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action
```

Comment:
This blew my mind 🤯

Reply:
@Kubitomakita From the mother of all GHA configs: https://github.com/ergebnis/php-library-template/tree/main/.github/workflows